molecule/default/tests/test_default.py

```diff
@@ -0,0 +1,26 @@
+import os
+import pytest
+
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
+
+
+@pytest.mark.parametrize('pkg', [
```

Comment:
This test isn't needed here, since this is already tested in the `ansible-role-logrotate` dependency.  It doesn't hurt, but I think it makes sense to remove it.  Feel free to disagree and take no action if you want, since this isn't an important issue at all.

Reply:
I had left it in there mainly because I wanted to ensure that the dependency was indeed being executed, but I see your point.  The same test was also left in [ansible-role-cyhy-logrotate](https://github.com/dhs-ncats/ansible-role-cyhy-logrotate/blob/develop/molecule/default/tests/test_default.py#L10).  I'm just going to leave it in there for now.

Reply:
OK